### DISCUSSION TITLE
Open SCAD data set

### DISCUSSION THREAD

**Discussion by simonsterer061** [LAION-AI/Open-Assistant#3445] [Ideas]
"OpenSCAD is not an interactive modeller. Instead, it is something like a 3D-compiler that reads in a script file that describes the object and renders the 3D model from this script file. This gives you (the designer) full control over the modelling process and enables you to easily change any step in the modelling process or make designs that are defined by configurable parameters." from the openSCAD website 
I have a lot of files in OpenSCAD I can label them all and add comments if needed i, but I don't know how to make a data set using hugging face so I will need some help with that

**Comment by simonsterer061** [comment]
This is the data that I have gathered so far, this doc will be continually updated as I get further in
https://docs.google.com/document/d/1a4xyzjQJQF7Vi5GcFi8n_6PdhnS548STmp9M_X5KE5c/edit?usp=sharing
it might be helpful to have other people add some requests, just remember that it needs to remain simple and not complex

**Reply by AraRL** [reply]
Hi @simonsterer061 the link seems broken. Do you still have the openscad files?

**Reply by itbear-zane** [reply]
Hi, @simonsterer061 . This data seems be deleted.

**Reply by Hanzz89** [reply]
😶

**Comment by mm12** [comment]
Yes please

**Comment by huu4ontocord** [comment]
thank you. we can help you convert your data into parquet or jsonl when the time comes.

**Comment by huu4ontocord** [comment]
Hi @simonsterer061 - are you at a stopping point and we can get someone to convert the google doc into a josnl file? Unless you want to take a crack at it. just let us know :)
